napari/_qt/qt_resources/__init__.py

```diff
@@ -156,9 +40,9 @@ def get_stylesheet(
     css : str
         The combined stylesheet.
     """
-    resources_dir = abspath(dirname(__file__))
+    resources_dir = os.path.abspath(os.path.dirname(__file__))
     stylesheet = ''
-    for file in sorted(glob(join(resources_dir, 'styles/*.qss'))):
+    for file in sorted(glob(os.path.join(resources_dir, 'styles/*.qss'))):
```

Comment:
would the forward slash work on windows ? I know it was already there; just wondering.

Reply:
good point!  seems to work fine though.  this is a function that gets used every time napari launches, so we'd have no styling on windows otherwise...